data/src/gamepad.js

```diff
@@ -47,7 +47,7 @@ class GamepadHandler {
                     id: oldGamepad.id
                 }
                 hasGamepad = true;
-                
```

Comment:
This file shouldnt be directly changed, its a module: https://github.com/ethanaobrien/Gamepad